src/authn/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
```

Comment:
I _think_ you might be able to just use `export * from './authn'` here :)

Reply:
Though I'm wondering whether this file is even necessary, given that the rest seems to refer to `authn/authn` directly?

Reply:
We want to export all methods as properties to `authn`-object - makes more sense when we get `authnWidget` and `authnControl` modules as welll.

Reply:
Maybe call this authnControl